features/step_definitions/user_steps.rb

```diff
@@ -402,3 +406,7 @@
   link = user_path(user)
   page.find(:css, %Q{a[href="#{link}?tab=activity"]}).trigger('click')
 end
+
+Then(/^I should not exist as a user$/) do
+  expect(@user).to be_nil
```

Comment:
hi @tansaku.  I think this is ok.  and I'm not sure how to reload the user given that @user has been destroyed by the delete_user method.